### DISCUSSION TITLE
utf-8 ansi bug

### DISCUSSION THREAD

**Discussion by ablemor** [qaisjp/mta-amx#77] [General]
hello guys, how can i fix ansi/utf-8 bug?

**Comment by Disinterpreter** [comment]
What do you mean? Could you give an example of your problem?

**Reply by ablemor** [reply]
example i write, Á, Ő, Ű, etc... I got random character... So this amx module cant handle ansi

And btw how to compile amx compatibility layer?

**Reply by Disinterpreter** [reply]
try to do it after space symbol
https://github.com/multitheftauto/amx/issues/61

**Reply by ablemor** [reply]
> try to do it after space symbol #61

after??

**Reply by Disinterpreter** [reply]
> 
> 
> > try to do it after space symbol #61
> 
> after??

https://github.com/multitheftauto/amx/issues/61#issuecomment-650575058

**Comment by ablemor** [comment]
heelpp

**Reply by ablemor** [reply]
done
